blogposts/announcing-sourcegraph-3.14.md

```diff
@@ -0,0 +1,258 @@
+* The internal indexed and unindexed search services are returning bad responses.
+* Gitserver may be under heavy load due to many concurrent command executions or under-provisioning.
+
+## Estimate required resources for your Sourcegraph instance
+
+TODO: screenshot
+
+The new [resource estimator](https://docs.sourcegraph.com/admin/install/resource_estimator) helps admins of new and growing Sourcegraph instances determine the right resources and deployment strategy for their custom scenario. CPU and memory needs are calculated for each service based on the number of repositories, users, large monorepos, and engagement rate of your users.
+
+## Repository management now can exclude GitHub forks/archives 
+
+Some organizations do not want to include any forks or archived repositories in their Sourcegraph instance. Admins can now exclude GitHub forks and/or archived repositories from the set of repositories being mirrored in Sourcegraph. The GitHub repository management can be updated with `"exclude": [{"forks": true}]` or `"exclude": [{"archived": true}]` to utilize this feature.
+
+## LSIF improvements to help advertise?
+
+Making the setup process better, links to browsing on open source projects. TBD - waiting on results from this week.
+
+## Aggregated search data added to Sourcegraph pings
+
+Sourcegraph collects a small amount of high-level and aggregate data from each instance. See the complete list of the [data we collect](https://docs.sourcegraph.com/admin/pings), and our [ping philosophy and rules](https://about.sourcegraph.com/handbook/engineering/adding_ping_data#ping-philosophy). New usage statistics—specifically, aggregate search latencies, aggregated counts of users and searches for search mode types, and aggregated counts of search filter usage will be sent to Sourcegraph via pings by default. The aggregated event count metrics can be disabled via the site admin flag `disableNonCriticalTelemetry`.
+
+## Experimental smart search bar
+
+TODO: Video
+
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/{ID}?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/{ID}" target="_blank">View on Vimeo</a></p>
+</p>
+-->
+
+In Sourcegraph 3.13 we introduced the smart search bar as an experimental feature. Many improvements have been made, though it will stay marked as experimental as we continue to solicit feedback. If you or your team are interested in trying this feature, add the following setting to your user, organization, or global settings on your Sourcegraph instance: `{ "experimentalFeatures": { "smartSearchField": true } }`. Tweet [@srcgraph](https://twitter.com/srcgraph) or email feedback@sourcegraph.com with your thoughts.
```

Comment:
```suggestion
In Sourcegraph 3.13, we introduced the smart search bar as an experimental feature. Many improvements have been made, but it will stay marked as experimental as we continue to solicit feedback. If you or your team are interested in trying this feature, add the following setting to your user, organization, or global settings on your Sourcegraph instance: `{ "experimentalFeatures": { "smartSearchField": true } }`. Tweet [@srcgraph](https://twitter.com/srcgraph) or email feedback@sourcegraph.com with your feedback.
```